aws/sdk/examples/apigateway/Cargo.toml

```diff
@@ -9,5 +9,6 @@ edition = "2018"
 aws-config = { path = "../../build/aws-sdk/aws-config" }
 aws-sdk-apigateway = { path = "../../build/aws-sdk/apigateway", package = "aws-sdk-apigateway" }
 tokio = { version = "1", features = ["full"] }
+actix-rt = "*"
```

Comment:
We have been using the Tokio runtime for async unit tests and we should be consistent here. Especially since some of our features rely on the Tokio runtime (until aws-sdk-rust#52 is implemented).

It's pretty easy to switch over:
```rust
#[tokio::test]
async fn test_get_rest_apis() {
    let shared_config = aws_config::load_from_env().await;
    let client = Client::new(&shared_config);

    client.get_rest_apis();
}
```

Reply:
Apparently this test doesn't do what I thought it was doing, so I'm removing it.